client/src/components/Headers/Navbar.tsx

```diff
@@ -0,0 +1,99 @@
+          {
+            landingPage &&
+            <>
+            <li>
+              <ScrollLink to='/' smooth duration={500}>
+                <p className="block py-2 px-3 text-white bg-blue-700 rounded md:bg-transparent md:text-blue-700 md:p-0" aria-current="page">Home</p>
+              </ScrollLink>
+            </li>
+            <li>
+              <ScrollLink to='about' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>About</p>
+              </ScrollLink>
+            </li>
+            {/* <li>
+              <ScrollLink to='' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Services</p>
+                </ScrollLink>
+              </li> */}
+            <li>
+              <ScrollLink to='contact' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Contact</p>
+              </ScrollLink>
+            </li>
+            <li>
+              <Link href='/auth/register'>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Register</p>
+              </Link>
+            </li>
+            <li>
+              <Link data-testid="Login-link" href='/auth/login'>
+                <button className='border-blue-700 border hover:bg-blue-700 hover:text-white hover:ring rounded px-3 py-1 text-blue-700 bg-transparent'>
+                  Login
+                </button>
+              </Link>
+            </li>
+              </>}
+            <li>
+              <button className='p-1 md:mt-0 mt-2' onClick={toggleTheme}>
+                {
+                  theme === 'light' && 
+                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M12 10.999c1.437.438 2.562 1.564 2.999 3.001.44-1.437 1.565-2.562 3.001-3-1.436-.439-2.561-1.563-3.001-3-.437 1.436-1.562 2.561-2.999 2.999zm8.001.001c.958.293 1.707 1.042 2 2.001.291-.959 1.042-1.709 1.999-2.001-.957-.292-1.707-1.042-2-2-.293.958-1.042 1.708-1.999 2zm-1-9c-.437 1.437-1.563 2.562-2.998 3.001 1.438.44 2.561 1.564 3.001 3.002.437-1.438 1.563-2.563 2.996-3.002-1.433-.437-2.559-1.564-2.999-3.001zm-7.001 22c-6.617 0-12-5.383-12-12s5.383-12 12-12c1.894 0 3.63.497 5.37 1.179-2.948.504-9.37 3.266-9.37 10.821 0 7.454 5.917 10.208 9.37 10.821-1.5.846-3.476 1.179-5.37 1.179z"/></svg>
```

Comment:
As discussed svg's should be in icons folder and used as a component. Reason to maintain folder structure.